packages/python/scripts/typecheck-or-skip.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+# Prefer uv (dev env), then python3 -m mypy when mypy is installed. No-op otherwise.
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)